### PR TITLE
fix(issues):repair issues

### PR DESCRIPTION
**Pull request by wzh425**
fix：repair issues

### CODE REVIEW

File: src/Web/Masa.Mc.Web.Admin/Components/Pickers/DateRangePicker.razor [hunk 1]
```diff
@@ -9,7 +9,7 @@
             </div>
         </ActivatorContent>
         <ChildContent>
-            <MDatePicker Value=_StartTime Max=_EndTime ValueChanged=UpdateStartTimeAsync TValue="DateOnly?" OnInput="()=> StartTimeVisible = false"></MDatePicker>
+            <DatePicker Value=_StartTime Max=_EndTime ValueChanged=UpdateStartTimeAsync TValue="DateOnly?" OnInput="()=> StartTimeVisible = false"></DatePicker>
```
Comment (doddgu) [inline src/Web/Masa.Mc.Web.Admin/Components/Pickers/DateRangePicker.razor:12]: Why use DatePicker?

File: src/Web/Masa.Mc.Web.Admin/Pages/Channels/Modules/ChannelCreateModal.razor [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿@inherits AdminCompontentBase
 <div @ref="Ref">
-    <McModal Value="@_visible" ValueChanged="HandleVisibleChanged" Title="@T("Permission.CreateChannel")" Width="770" Height="792">
+    <McModal Value="@_visible" ValueChanged="HandleVisibleChanged" Title="@T("Permission.CreateChannel")" ContentStyle="height: 597px;">
```
Comment (doddgu) [inline src/Web/Masa.Mc.Web.Admin/Pages/Channels/Modules/ChannelCreateModal.razor:3]: Why you have McModal?

File: src/Web/Masa.Mc.Web.Admin/wwwroot/i18n/en-US.json [hunk 3]
```diff
@@ -29,12 +29,13 @@
   "Permission.EditChannel": "Edit channel",
   "SelectChannelType": "Select channel type",
   "ChannelConfiguration": "Channel configuration",
-  "EmailChannelOptions.UserName": "邮箱账号",
-  "EmailChannelOptions.Password": "密码",
-  "EmailChannelOptions.Smtp": "SMTP服务器",
+  "EmailChannelOptions.UserName": "UserName",
```
Comment (doddgu) [inline src/Web/Masa.Mc.Web.Admin/wwwroot/i18n/en-US.json:32]: User Name

File: src/Web/Masa.Mc.Web.Admin/wwwroot/i18n/en-US.json [hunk 4]
```diff
@@ -29,12 +29,13 @@
...
+  "EmailChannelOptions.Password": "Password",
+  "EmailChannelOptions.Smtp": "SMTPSmtp",
```
Comment (doddgu) [inline src/Web/Masa.Mc.Web.Admin/wwwroot/i18n/en-US.json:34]: SMTPSmtp?

File: src/Web/Masa.Mc.Web.Admin/wwwroot/i18n/en-US.json [hunk 5]
```diff
@@ -29,12 +29,13 @@
...
   "EmailChannelOptions.Ssl": "Ssl",
-  "EmailChannelOptions.Port": "端口",
-  "WebsiteMessageChannelOptions.Environment": "环境",
+  "EmailChannelOptions.Port": "Port",
+  "EmailChannelOptions.FromNickName": "FromNickName",
```
Comment (doddgu) [inline src/Web/Masa.Mc.Web.Admin/wwwroot/i18n/en-US.json:37]: From Nick Name

File: src/Web/Masa.Mc.Web.Admin/wwwroot/i18n/en-US.json [hunk 6]
```diff
@@ -29,12 +29,13 @@
...
+  "WebsiteMessageChannelOptions.Environment": "Environment",
   "SmsChannelOptions.AccessKeyId": "AccessKeyId",
```
Comment (doddgu) [inline src/Web/Masa.Mc.Web.Admin/wwwroot/i18n/en-US.json:39]: Access Key Id

File: src/Web/Masa.Mc.Web.Admin/wwwroot/i18n/en-US.json [hunk 7]
```diff
@@ -29,12 +29,13 @@
...
   "SmsChannelOptions.AccessKeySecret": "AccessKeySecret",
```
Comment (doddgu) [inline src/Web/Masa.Mc.Web.Admin/wwwroot/i18n/en-US.json:40]: Access Key Secret

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Web/Masa.Mc.Web.Admin/Components/Pickers/DateRangePicker.razor (hunk 1 above)
Comment (wzh425) [reply src/Web/Masa.Mc.Web.Admin/Components/Pickers/DateRangePicker.razor:12]: 多了重置和今日的按钮

Comment (doddgu) [reply src/Web/Masa.Mc.Web.Admin/Components/Pickers/DateRangePicker.razor:12]: Is DatePicker from MASA.Stack.Component?

File: src/Web/Masa.Mc.Web.Admin/Pages/Channels/Modules/ChannelCreateModal.razor (hunk 2 above)
Comment (wzh425) [reply src/Web/Masa.Mc.Web.Admin/Pages/Channels/Modules/ChannelCreateModal.razor:3]: 为了还原设计的弹窗 我将迁移到MASA.Stack.Components 并改用PModal 实现

File: src/Web/Masa.Mc.Web.Admin/Components/Pickers/DateRangePicker.razor (hunk 1 above)
Comment (wzh425) [reply src/Web/Masa.Mc.Web.Admin/Components/Pickers/DateRangePicker.razor:12]: 是的

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**